services/web/client/source/class/qxapp/component/filter/UIFilter.js

```diff
@@ -32,8 +32,10 @@ qx.Class.define("qxapp.component.filter.UIFilter", {
    */
   construct: function(filterId, groupId) {
     this.base(arguments);
-    this.setFilterId(filterId);
-    this.setGroupId(groupId);
+    this.set({
+      filterId,
```

Comment:
```
filterId: filerId,
groupId: groupId
```

Reply:
```
{
   filterId: filterId
}
```
is equivalent to
```
{
   filterId
}
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015